packages/bot-handler-avs/package.json

```diff
@@ -1,8 +1,7 @@
 {
   "dependencies": {
     "@types/webrtc": "0.0.26",
-    "@wireapp/avs": "6.0.46",
-    "wrtc": "0.4.5"
+    "@wireapp/avs": "6.0.46"
```

Comment:
Let's update to the latest AVS 6.2 here.

Reply:
I'd rather not change other dependencies here.